pkg/torcx/propagate.go

```diff
@@ -242,7 +243,22 @@ func symlinkUnitAsset(applyCfg *ApplyConfig, unitsDir string, asset string) erro
 				// Do not overwrite previous assets
 				return nil
 			}
-			return os.Symlink(path, hostPath)
+			fpDst, err := os.Create(hostPath)
+			if err != nil {
+				return errors.Wrapf(err, "error creating %q", hostPath)
+			}
+			fpSrc, err := os.Open(path)
+			if err != nil {
+				return errors.Wrapf(err, "error opening %q", path)
+			}
+			copied, err := io.Copy(fpDst, fpSrc)
+			if err != nil {
+				return errors.Wrapf(err, "error copying to %q", hostPath)
+			}
+			if copied != inInfo.Size() {
```

Comment:
Does `io.Copy` not return `io.ErrShortWrite` in some cases? Is this just being extra defensive?

I'd think this can just be `_, err = io.Copy...; return err`